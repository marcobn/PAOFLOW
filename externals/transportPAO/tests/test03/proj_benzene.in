
&projwfc
  prefix='benzene'
  outdir='./SCRATCH'
  filpdos='./SCRATCH/benzene'
  lwrite_overlaps = .TRUE.
  !lbinary_data = .TRUE.
/
