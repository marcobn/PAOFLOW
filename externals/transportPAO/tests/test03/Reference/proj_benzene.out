
     Program PROJWFC v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:37: 8 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1s, nr2s, nr3s values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      =  SLA  LYP  B88  BLYP ( 1 3 1 3 0)
     EXX-fraction              =        0.00
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        2148    2148    538               183436   183436   22924
     Max        2149    2149    540               183439   183439   22925
     Sum       12893   12893   3233              1100629  1100629  137545
 

     Gaussian broadening (read from file): ngauss,degauss=   0    0.010000


     Calling projwave .... 

     Atomic states used for projection
     (read from pseudopotential files):

     state #   1: atom   1 (H  ), wfc  1 (l=0 m= 1)
     state #   2: atom   2 (C  ), wfc  1 (l=0 m= 1)
     state #   3: atom   2 (C  ), wfc  2 (l=1 m= 1)
     state #   4: atom   2 (C  ), wfc  2 (l=1 m= 2)
     state #   5: atom   2 (C  ), wfc  2 (l=1 m= 3)
     state #   6: atom   3 (H  ), wfc  1 (l=0 m= 1)
     state #   7: atom   4 (C  ), wfc  1 (l=0 m= 1)
     state #   8: atom   4 (C  ), wfc  2 (l=1 m= 1)
     state #   9: atom   4 (C  ), wfc  2 (l=1 m= 2)
     state #  10: atom   4 (C  ), wfc  2 (l=1 m= 3)
     state #  11: atom   5 (H  ), wfc  1 (l=0 m= 1)
     state #  12: atom   6 (C  ), wfc  1 (l=0 m= 1)
     state #  13: atom   6 (C  ), wfc  2 (l=1 m= 1)
     state #  14: atom   6 (C  ), wfc  2 (l=1 m= 2)
     state #  15: atom   6 (C  ), wfc  2 (l=1 m= 3)
     state #  16: atom   7 (H  ), wfc  1 (l=0 m= 1)
     state #  17: atom   8 (C  ), wfc  1 (l=0 m= 1)
     state #  18: atom   8 (C  ), wfc  2 (l=1 m= 1)
     state #  19: atom   8 (C  ), wfc  2 (l=1 m= 2)
     state #  20: atom   8 (C  ), wfc  2 (l=1 m= 3)
     state #  21: atom   9 (H  ), wfc  1 (l=0 m= 1)
     state #  22: atom  10 (C  ), wfc  1 (l=0 m= 1)
     state #  23: atom  10 (C  ), wfc  2 (l=1 m= 1)
     state #  24: atom  10 (C  ), wfc  2 (l=1 m= 2)
     state #  25: atom  10 (C  ), wfc  2 (l=1 m= 3)
     state #  26: atom  11 (H  ), wfc  1 (l=0 m= 1)
     state #  27: atom  12 (C  ), wfc  1 (l=0 m= 1)
     state #  28: atom  12 (C  ), wfc  2 (l=1 m= 1)
     state #  29: atom  12 (C  ), wfc  2 (l=1 m= 2)
     state #  30: atom  12 (C  ), wfc  2 (l=1 m= 3)

 k =   0.0000000000  0.0000000000  0.0000000000
==== e(   1) =   -20.80257 eV ==== 
     psi = 0.121*[#  12]+0.121*[#   7]+0.121*[#  17]+0.121*[#   2]+0.121*[#  22]+
          +0.121*[#  27]+0.026*[#  14]+0.026*[#  29]+0.020*[#   5]+0.020*[#  10]+
          +0.020*[#  20]+0.020*[#  25]+0.017*[#   1]+0.017*[#   6]+0.017*[#  11]+
          +0.017*[#  16]+0.017*[#  21]+0.017*[#  26]+0.007*[#   4]+0.007*[#   9]+
          +0.006*[#  19]+0.007*[#  24]+
    |psi|^2 = 0.981
==== e(   2) =   -18.08806 eV ==== 
     psi = 0.198*[#   2]+0.198*[#  17]+0.084*[#  27]+0.084*[#  12]+0.056*[#   9]+
          +0.056*[#  24]+0.049*[#  15]+0.048*[#  30]+0.040*[#   6]+0.040*[#  21]+
          +0.024*[#   7]+0.024*[#  22]+0.017*[#   1]+0.017*[#  10]+0.017*[#  16]+
          +0.017*[#  25]+0.005*[#  11]+0.005*[#  26]+0.002*[#   5]+0.002*[#  20]+
          +0.001*[#   4]+0.001*[#  19]+
    |psi|^2 = 0.983
==== e(   3) =   -18.08422 eV ==== 
     psi = 0.180*[#  22]+0.180*[#   7]+0.120*[#  27]+0.120*[#  12]+0.061*[#   4]+
          +0.061*[#  19]+0.036*[#  11]+0.036*[#  26]+0.034*[#  15]+0.034*[#  30]+
          +0.024*[#   1]+0.024*[#  16]+0.019*[#   5]+0.019*[#  20]+0.006*[#   2]+
          +0.006*[#  17]+0.006*[#   9]+0.006*[#  24]+0.004*[#  10]+0.004*[#  25]+
          +0.001*[#   6]+0.001*[#  21]+
    |psi|^2 = 0.983
==== e(   4) =   -14.51448 eV ==== 
     psi = 0.140*[#  15]+0.140*[#  30]+0.089*[#   2]+0.089*[#  17]+0.070*[#   9]+
          +0.070*[#  24]+0.064*[#  22]+0.064*[#   7]+0.045*[#   6]+0.045*[#  21]+
          +0.039*[#   4]+0.039*[#  19]+0.033*[#  26]+0.033*[#  11]+0.004*[#   5]+
          +0.004*[#  20]+0.002*[#  12]+0.002*[#  27]+0.001*[#   1]+0.001*[#  16]+
    |psi|^2 = 0.975
==== e(   5) =   -14.51391 eV ==== 
     psi = 0.101*[#  12]+0.101*[#  27]+0.075*[#   4]+0.075*[#  19]+0.055*[#  10]+
          +0.055*[#  25]+0.052*[#  20]+0.052*[#   1]+0.052*[#   5]+0.052*[#  16]+
          +0.043*[#   9]+0.043*[#  24]+0.039*[#   7]+0.039*[#  22]+0.026*[#  14]+
          +0.026*[#  29]+0.020*[#  11]+0.020*[#  26]+0.014*[#   2]+0.014*[#  17]+
          +0.007*[#   6]+0.007*[#  21]+0.003*[#  15]+0.003*[#  30]+
    |psi|^2 = 0.975
==== e(   6) =   -12.56709 eV ==== 
     psi = 0.111*[#  14]+0.111*[#  29]+0.084*[#  20]+0.083*[#  10]+0.083*[#   5]+
          +0.083*[#  25]+0.043*[#  11]+0.043*[#   1]+0.043*[#  21]+0.043*[#  16]+
          +0.043*[#   6]+0.043*[#  26]+0.028*[#   9]+0.028*[#  19]+0.028*[#  24]+
          +0.028*[#   4]+0.006*[#   7]+0.006*[#   2]+0.006*[#  12]+0.006*[#  17]+
          +0.006*[#  22]+0.006*[#  27]+
    |psi|^2 = 0.963
==== e(   7) =   -10.99568 eV ==== 
     psi = 0.070*[#   6]+0.070*[#   1]+0.070*[#  16]+0.070*[#  21]+0.070*[#  26]+
          +0.069*[#  11]+0.059*[#  14]+0.059*[#  29]+0.045*[#   5]+0.045*[#  25]+
          +0.044*[#  10]+0.044*[#  20]+0.035*[#   2]+0.035*[#  22]+0.035*[#  12]+
          +0.035*[#  27]+0.035*[#   7]+0.035*[#  17]+0.015*[#   4]+0.015*[#  19]+
          +0.015*[#  24]+0.014*[#   9]+
    |psi|^2 = 0.986
==== e(   8) =   -10.54651 eV ==== 
     psi = 0.164*[#  15]+0.163*[#  30]+0.123*[#   9]+0.123*[#  24]+0.123*[#   4]+
          +0.123*[#  19]+0.041*[#  20]+0.041*[#   5]+0.041*[#  10]+0.040*[#  25]+
    |psi|^2 = 0.982
==== e(   9) =    -9.96878 eV ==== 
     psi = 0.174*[#  20]+0.174*[#   5]+0.108*[#  29]+0.108*[#  14]+0.077*[#  21]+
          +0.077*[#   6]+0.046*[#  19]+0.045*[#   4]+0.038*[#   1]+0.038*[#  16]+
          +0.025*[#  24]+0.024*[#   9]+0.007*[#  11]+0.007*[#  26]+0.006*[#  15]+
          +0.006*[#  30]+0.005*[#  10]+0.005*[#  25]+0.002*[#   2]+0.002*[#  17]+
          +0.001*[#  12]+0.001*[#  27]+
    |psi|^2 = 0.980
==== e(  10) =    -9.96395 eV ==== 
     psi = 0.171*[#  10]+0.171*[#  25]+0.123*[#  29]+0.123*[#  14]+0.075*[#  11]+
          +0.075*[#  26]+0.044*[#   1]+0.043*[#  16]+0.042*[#   9]+0.041*[#  24]+
          +0.020*[#   4]+0.020*[#  19]+0.005*[#  15]+0.005*[#  30]+0.004*[#   6]+
          +0.004*[#  21]+0.002*[#   7]+0.002*[#  22]+0.002*[#   5]+0.002*[#  20]+
          +0.001*[#  12]+0.001*[#  27]+
    |psi|^2 = 0.980
==== e(  11) =    -8.59367 eV ==== 
     psi = 0.163*[#  13]+0.163*[#   8]+0.163*[#  18]+0.163*[#   3]+0.163*[#  23]+
          +0.163*[#  28]+
    |psi|^2 = 0.978
==== e(  12) =    -7.99936 eV ==== 
     psi = 0.106*[#  30]+0.105*[#  15]+0.105*[#  25]+0.104*[#   5]+0.104*[#  10]+
          +0.103*[#  20]+0.093*[#   6]+0.093*[#  21]+0.041*[#  26]+0.041*[#  11]+
          +0.013*[#   9]+0.013*[#  24]+0.013*[#  14]+0.013*[#  29]+0.013*[#   4]+
          +0.013*[#  19]+0.011*[#   1]+0.011*[#  16]+
    |psi|^2 = 0.993
==== e(  13) =    -7.99894 eV ==== 
     psi = 0.105*[#   4]+0.105*[#  19]+0.105*[#   9]+0.105*[#  24]+0.103*[#  14]+
          +0.103*[#  29]+0.086*[#  16]+0.085*[#   1]+0.056*[#  26]+0.055*[#  11]+
          +0.013*[#   5]+0.013*[#  20]+0.013*[#  30]+0.013*[#  15]+0.012*[#  25]+
          +0.012*[#  10]+0.003*[#   6]+0.003*[#  21]+
    |psi|^2 = 0.993
==== e(  14) =    -5.89117 eV ==== 
     psi = 0.320*[#   3]+0.320*[#  18]+0.143*[#  28]+0.143*[#  13]+0.035*[#   8]+
          +0.035*[#  23]+
    |psi|^2 = 0.997
==== e(  15) =    -5.88846 eV ==== 
     psi = 0.298*[#  23]+0.297*[#   8]+0.189*[#  28]+0.189*[#  13]+0.012*[#   3]+
          +0.012*[#  18]+
    |psi|^2 = 0.997
==== e(  16) =    -0.84278 eV ==== 
     psi = 0.241*[#  23]+0.241*[#   8]+0.225*[#  28]+0.225*[#  13]+
    |psi|^2 = 0.932
==== e(  17) =    -0.84110 eV ==== 
     psi = 0.311*[#   3]+0.310*[#  18]+0.086*[#  28]+0.086*[#  13]+0.070*[#   8]+
          +0.070*[#  23]+
    |psi|^2 = 0.932
==== e(  18) =    -0.29755 eV ==== 
     psi = 0.038*[#   1]+0.038*[#  16]+0.038*[#   6]+0.038*[#  11]+0.038*[#  21]+
          +0.038*[#  26]+0.008*[#   2]+0.008*[#   7]+0.008*[#  12]+0.008*[#  17]+
          +0.008*[#  22]+0.008*[#  27]+0.004*[#  14]+0.004*[#  29]+0.003*[#   5]+
          +0.003*[#  10]+0.003*[#  20]+0.003*[#  25]+0.001*[#   4]+0.001*[#   9]+
          +0.001*[#  19]+0.001*[#  24]+
    |psi|^2 = 0.303
==== e(  19) =     0.30689 eV ==== 
     psi = 0.045*[#   1]+0.045*[#  16]+0.013*[#   6]+0.013*[#  21]+0.012*[#  11]+
          +0.012*[#  26]+0.010*[#  12]+0.010*[#  27]+0.006*[#  14]+0.006*[#  29]+
          +0.003*[#   5]+0.003*[#  10]+0.003*[#  20]+0.003*[#  25]+0.003*[#   2]+
          +0.003*[#  17]+0.003*[#   7]+0.003*[#  22]+
    |psi|^2 = 0.199
==== e(  20) =     0.30743 eV ==== 
     psi = 0.035*[#  11]+0.036*[#  26]+0.035*[#   6]+0.035*[#  21]+0.008*[#   7]+
          +0.008*[#  22]+0.008*[#   2]+0.008*[#  17]+0.003*[#   4]+0.003*[#   9]+
          +0.003*[#  19]+0.003*[#  15]+0.003*[#  24]+0.003*[#  30]+0.002*[#   5]+
          +0.002*[#  10]+0.002*[#  20]+0.002*[#  25]+
    |psi|^2 = 0.199
==== e(  21) =     0.36439 eV ==== 
     psi = 0.013*[#   1]+0.013*[#  16]+0.012*[#   6]+0.012*[#  11]+0.012*[#  21]+
          +0.012*[#  26]+0.003*[#   2]+0.003*[#   7]+0.003*[#  12]+0.003*[#  17]+
          +0.003*[#  22]+0.003*[#  27]+0.002*[#  14]+0.002*[#  29]+0.001*[#   5]+
          +0.001*[#  10]+0.001*[#  20]+0.001*[#  25]+
    |psi|^2 = 0.102
==== e(  22) =     0.54616 eV ==== 
     psi = 0.009*[#   1]+0.009*[#  16]+0.007*[#  12]+0.007*[#  27]+0.002*[#   6]+
          +0.002*[#  11]+0.002*[#  21]+0.002*[#  26]+0.002*[#   4]+0.002*[#   9]+
          +0.002*[#  19]+0.002*[#  24]+0.002*[#   2]+0.002*[#   7]+0.002*[#  17]+
          +0.002*[#  22]+
    |psi|^2 = 0.059
==== e(  23) =     0.54992 eV ==== 
     psi = 
    |psi|^2 = 0.004
==== e(  24) =     0.63747 eV ==== 
     psi = 
    |psi|^2 = 0.001
==== e(  25) =     0.89705 eV ==== 
     psi = 0.017*[#   6]+0.017*[#  11]+0.017*[#  26]+0.017*[#  21]+0.014*[#   2]+
          +0.014*[#   7]+0.014*[#  17]+0.014*[#  22]+0.008*[#  15]+0.008*[#  30]+
          +0.002*[#   5]+0.002*[#  10]+0.002*[#  20]+0.002*[#  25]+
    |psi|^2 = 0.151
==== e(  26) =     0.95987 eV ==== 
     psi = 0.020*[#   1]+0.020*[#  16]+0.005*[#  12]+0.005*[#  27]+0.003*[#  14]+
          +0.003*[#  29]+0.002*[#   6]+0.002*[#  11]+0.002*[#  21]+0.002*[#  26]+
    |psi|^2 = 0.066
==== e(  27) =     0.96172 eV ==== 
     psi = 0.011*[#   6]+0.011*[#  11]+0.011*[#  21]+0.011*[#  26]+0.002*[#   2]+
          +0.002*[#   7]+0.002*[#  17]+0.002*[#  22]+0.001*[#   4]+0.001*[#   9]+
          +0.001*[#  19]+0.001*[#  24]+
    |psi|^2 = 0.063
==== e(  28) =     1.11731 eV ==== 
     psi = 0.010*[#   1]+0.010*[#  16]+0.008*[#  12]+0.008*[#  27]+0.002*[#   4]+
          +0.002*[#   9]+0.002*[#  19]+0.002*[#  24]+0.002*[#   6]+0.002*[#  11]+
          +0.002*[#  21]+0.002*[#  26]+0.002*[#   2]+0.002*[#   7]+0.002*[#  17]+
          +0.002*[#  22]+
    |psi|^2 = 0.060
==== e(  29) =     1.11897 eV ==== 
     psi = 
    |psi|^2 = 0.004
==== e(  30) =     1.12721 eV ==== 
     psi = 0.009*[#   6]+0.009*[#  11]+0.009*[#  21]+0.009*[#  26]+0.007*[#   1]+
          +0.007*[#  16]+0.003*[#   2]+0.003*[#   7]+0.003*[#  17]+0.003*[#  22]+
          +0.002*[#  12]+0.002*[#  27]+
    |psi|^2 = 0.071
==== e(  31) =     1.16424 eV ==== 
     psi = 
    |psi|^2 = 0.000
==== e(  32) =     1.16486 eV ==== 
     psi = 
    |psi|^2 = 0.000
==== e(  33) =     1.17896 eV ==== 
     psi = 
    |psi|^2 = 0.008
==== e(  34) =     1.23829 eV ==== 
     psi = 0.002*[#   6]+0.002*[#  11]+0.002*[#  21]+0.002*[#  26]+
    |psi|^2 = 0.014
==== e(  35) =     1.25262 eV ==== 
     psi = 
    |psi|^2 = 0.001
==== e(  36) =     1.26430 eV ==== 
     psi = 0.001*[#  13]+0.001*[#  28]+
    |psi|^2 = 0.003
==== e(  37) =     1.62468 eV ==== 
     psi = 0.006*[#  11]+0.006*[#  26]+0.005*[#   6]+0.005*[#   7]+0.005*[#  21]+
          +0.005*[#  22]+0.005*[#   2]+0.005*[#  17]+0.003*[#  15]+0.003*[#  30]+
    |psi|^2 = 0.050
==== e(  38) =     1.63539 eV ==== 
     psi = 0.006*[#   6]+0.006*[#  21]+0.006*[#  11]+0.006*[#  26]+0.005*[#   1]+
          +0.005*[#  16]+0.004*[#   2]+0.004*[#  12]+0.004*[#  17]+0.004*[#  27]+
          +0.004*[#   7]+0.004*[#  22]+
    |psi|^2 = 0.060
==== e(  39) =     1.71536 eV ==== 
     psi = 0.014*[#   1]+0.014*[#  16]+0.004*[#  12]+0.004*[#  27]+0.002*[#  14]+
          +0.002*[#  29]+
    |psi|^2 = 0.041
==== e(  40) =     1.72653 eV ==== 
     psi = 0.005*[#   6]+0.005*[#  11]+0.005*[#  21]+0.005*[#  26]+0.001*[#   2]+
          +0.001*[#   7]+0.001*[#  17]+0.001*[#  22]+
    |psi|^2 = 0.029
==== e(  41) =     1.75869 eV ==== 
     psi = 
    |psi|^2 = 0.002
==== e(  42) =     1.76289 eV ==== 
     psi = 
    |psi|^2 = 0.000
==== e(  43) =     1.81068 eV ==== 
     psi = 
    |psi|^2 = 0.002
==== e(  44) =     1.82763 eV ==== 
     psi = 0.001*[#   3]+0.001*[#   8]+0.001*[#  18]+0.001*[#  23]+
    |psi|^2 = 0.006
==== e(  45) =     1.85894 eV ==== 
     psi = 0.023*[#  12]+0.023*[#  27]+0.022*[#   1]+0.022*[#  16]+0.006*[#   4]+
          +0.006*[#   9]+0.006*[#  19]+0.006*[#  24]+0.006*[#   2]+0.006*[#   7]+
          +0.006*[#  17]+0.006*[#  22]+0.005*[#   6]+0.005*[#  11]+0.005*[#  21]+
          +0.005*[#  26]+0.001*[#   5]+0.001*[#  10]+0.001*[#  20]+0.001*[#  25]+
    |psi|^2 = 0.166
==== e(  46) =     1.86848 eV ==== 
     psi = 0.003*[#   6]+0.003*[#  11]+0.003*[#  21]+0.003*[#  26]+0.001*[#   1]+
          +0.001*[#  16]+
    |psi|^2 = 0.022
==== e(  47) =     2.14616 eV ==== 
     psi = 0.023*[#   6]+0.023*[#  11]+0.023*[#  21]+0.023*[#  26]+0.009*[#   2]+
          +0.009*[#   7]+0.009*[#  17]+0.009*[#  22]+0.002*[#   5]+0.002*[#  10]+
          +0.002*[#  20]+0.002*[#  25]+0.002*[#  12]+0.002*[#  27]+0.001*[#   1]+
          +0.001*[#  16]+
    |psi|^2 = 0.145
==== e(  48) =     2.21824 eV ==== 
     psi = 0.004*[#   6]+0.004*[#  11]+0.004*[#  21]+0.004*[#  26]+0.002*[#  15]+
          +0.002*[#  30]+
    |psi|^2 = 0.023
==== e(  49) =     2.25128 eV ==== 
     psi = 
    |psi|^2 = 0.001
==== e(  50) =     2.45711 eV ==== 
     psi = 0.004*[#   1]+0.004*[#  16]+0.002*[#  14]+0.002*[#  29]+
    |psi|^2 = 0.018
==== e(  51) =     2.54092 eV ==== 
     psi = 0.013*[#   2]+0.013*[#   7]+0.013*[#  17]+0.013*[#  22]+0.010*[#   6]+
          +0.010*[#  11]+0.010*[#  21]+0.010*[#  26]+0.008*[#  15]+0.008*[#  30]+
          +0.001*[#   4]+0.001*[#   9]+0.001*[#  19]+0.001*[#  24]+
    |psi|^2 = 0.116
==== e(  52) =     2.63185 eV ==== 
     psi = 0.008*[#   6]+0.008*[#  11]+0.008*[#  21]+0.008*[#  26]+0.004*[#   1]+
          +0.004*[#  16]+0.004*[#   2]+0.004*[#   7]+0.004*[#  17]+0.004*[#  22]+
          +0.002*[#  12]+0.002*[#  27]+
    |psi|^2 = 0.067
==== e(  53) =     2.64824 eV ==== 
     psi = 0.002*[#   1]+0.002*[#  16]+
    |psi|^2 = 0.011
==== e(  54) =     2.65071 eV ==== 
     psi = 0.002*[#   5]+0.002*[#  10]+0.002*[#  20]+0.002*[#  25]+0.002*[#   6]+
          +0.001*[#  11]+0.002*[#  21]+0.001*[#  26]+
    |psi|^2 = 0.015
==== e(  55) =     2.69503 eV ==== 
     psi = 0.017*[#   3]+0.017*[#   8]+0.017*[#  18]+0.017*[#  23]+0.016*[#  13]+
          +0.016*[#  28]+0.003*[#  12]+0.003*[#  27]+0.002*[#   1]+0.002*[#  16]+
    |psi|^2 = 0.120
==== e(  56) =     2.69819 eV ==== 
     psi = 0.024*[#   3]+0.024*[#   8]+0.024*[#  18]+0.024*[#  23]+0.022*[#  13]+
          +0.022*[#  28]+0.002*[#  12]+0.002*[#  27]+0.002*[#   1]+0.002*[#  16]+
    |psi|^2 = 0.152
==== e(  57) =     2.75489 eV ==== 
     psi = 
    |psi|^2 = 0.000
==== e(  58) =     2.77312 eV ==== 
     psi = 0.024*[#   1]+0.024*[#  16]+0.007*[#  14]+0.007*[#  29]+0.006*[#  12]+
          +0.006*[#  27]+
    |psi|^2 = 0.079
==== e(  59) =     2.80574 eV ==== 
     psi = 0.008*[#   6]+0.008*[#  11]+0.008*[#  21]+0.008*[#  26]+0.002*[#   5]+
          +0.002*[#  10]+0.002*[#  20]+0.002*[#  25]+0.002*[#   2]+0.002*[#   7]+
          +0.002*[#  17]+0.002*[#  22]+0.001*[#   4]+0.001*[#   9]+0.001*[#  19]+
          +0.001*[#  24]+
    |psi|^2 = 0.051
==== e(  60) =     2.84293 eV ==== 
     psi = 0.065*[#  13]+0.065*[#  28]+0.063*[#   3]+0.063*[#   8]+0.063*[#  18]+
          +0.063*[#  23]+
    |psi|^2 = 0.381

Lowdin Charges: 

     Atom #   1: total charge =   0.8022, s =  0.8022, p =  0.0000, 
     Atom #   2: total charge =   4.1130, s =  0.9448, p =  3.1682, 
     Atom #   3: total charge =   0.8021, s =  0.8021, p =  0.0000, 
     Atom #   4: total charge =   4.1131, s =  0.9448, p =  3.1684, 
     Atom #   5: total charge =   0.8021, s =  0.8021, p =  0.0000, 
     Atom #   6: total charge =   4.1129, s =  0.9447, p =  3.1683, 
     Atom #   7: total charge =   0.8022, s =  0.8022, p =  0.0000, 
     Atom #   8: total charge =   4.1132, s =  0.9446, p =  3.1686, 
     Atom #   9: total charge =   0.8019, s =  0.8019, p =  0.0000, 
     Atom #  10: total charge =   4.1130, s =  0.9450, p =  3.1680, 
     Atom #  11: total charge =   0.8023, s =  0.8023, p =  0.0000, 
     Atom #  12: total charge =   4.1131, s =  0.9450, p =  3.1681, 
     Spilling Parameter:   0.0170
